Apps/AdminWebClient/src/appsettings.json

```diff
@@ -1,48 +1,48 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft":  "Warning"
-    }
-  },
-  "SwaggerSettings": {
-    "RoutePrefix": "swagger",
-    "Info": {
-      "Title": "Health Gateway WebClient Services documentation",
-      "Description": "Provides API documentation for Health Gateway WebClient Services.",
-      "License": {
-        "Name": "Apache 2.0",
-        "Url": "https://www.apache.org/licenses/LICENSE-2.0.html"
-      }
-    }
-  },
-  "OpenIdConnect": {
-    "SaveTokens": "false",
-    "GetClaimsFromUserInfoEndpoint": "true",
-    "SignedOutRedirectUri": "/signoff",
-    "ResponseType": "code",
-    "Scope": "openid profile email offline_access",
-    "RequireHttpsMetadata": "false"
-  },
-  "KeyCloak": {
```

Comment:
Were these removed on purpose?

Reply:
not removed, the changes are not showing side by side.